l10n.js

```diff
@@ -1,74 +1,17 @@
 var l10n = {
   "en": {
-    "language":"English",
-    "translator":" ",
-    "unique":"Dream up a character for this SDS!\n<br/><br/>Create a new character that could be used as a sprite in someones project. Your character could be anything; an orange dragon, a fluffy cloud, or even a bag of popcorn with a face!<br/><br/>Once you're done, give your character a personality! Write a short description about it, it can be anything; what it likes, hobbies, favorite color, etc.<br/><br/>Be sure to animate your character in some way to give it its own twist!<br/>",
-    "standard":"To have your project added to this studio, please post a comment with a link. You can find a link to your project by clicking the ‘Embed’ button on the project page. The link will be directly below the Facebook logo.\n        <br/>\n        Please submit a single project made no earlier than one month before the start of this Design studio. However, you can update your project, and it'll be eligible again. If you have an older project that would fit, add some new features and feel free to submit the updated version!\n        <br/>\n        All projects must meet the community guidelines and be appropriate for all ages!\n        <br/>\n        It may take some time for your projects to be added. Please be patient, we add projects oldest to newest; no need to double post!\n        <br/><br/>\n\n        SDS (Scratch Design Studio) is a themed studio where any Scratcher can submit a project that matches the current theme so long as it meets the requirements shown above.\n        <!-- No br here as it looks fine without -->\n        Check out this project to see how new SDS curators are selected: <a href=\"https://scratch.mit.edu/projects/55738732/\">https://scratch.mit.edu/projects/55738732/</a>\n        Please do not simply ask to be a curator!\n        <br/><br/>\n\n        Do you have an idea for a future design studio? Visit the studio linked below, read the studio description, and then submit your idea in the form of a project here: <a href=\"http://scratch.mit.edu/studios/93627/\">http://scratch.mit.edu/studios/93627/</a>\n        <br/><br/>\n\n        Questions about Scratch Design Studio? Check out the Scratch Wiki's article on it: <a href=\"http://wiki.scratch.mit.edu/wiki/SDS/\">http://wiki.scratch.mit.edu/wiki/SDS/</a>\n        <br/><br/>\n\n        Check out this account for new updates on the SDS, new SDS curators and more! <a href=\"https://scratch.mit.edu/users/SDS-Updates/\">@SDS-Updates</a>\n\n<br/",
-    "curators":"The SDS is curated by"
-  },
-  "es": {
```

Comment:
Whoa now! The 'standard' part of the translated descriptions are still very useful, we shouldn't be removing them! What we do is just change the 'unique' part to `Translation Needed!`